### DISCUSSION TITLE
I'm coding my first project using state adapt, could you give me any suggestions?

### DISCUSSION THREAD

**Discussion by daniromome** [state-adapt/state-adapt#51] [Show and tell]
I come from an NgRx background and it was a bit hard for me to understand/ go through the docs and make sense of them, this is with what I ended up with

[auth.service.ts](https://gist.github.com/daniromome/17cd4477fe24c697b16f91c44af7f2fc)
[auth.adapter.ts](https://gist.github.com/daniromome/166b2893fac5e71200b90cfa60d0357a)
[api.ts](https://gist.github.com/daniromome/4492244e0585bed840619201ebc1ba7c)

Could anyone give me any suggestions on how could I improve my code? Thanks in advance!

**Comment by mfp22** [comment]
Looks good to me.

I still need to make a tutorial. The docs are more for reference.

**Reply by daniromome** [reply]
Thanks for taking your time to look at it! Great work on the library :)
